com.hybris.hyeclipse.property.configuration/src/com/hybris/hyeclipse/property/configuration/handlers/SavePropertyHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.hybris.hyeclipse.property.configuration.handlers;
+
+import java.util.Map;
+
+import org.eclipse.jface.dialogs.MessageDialog;
+
+import com.hybris.hyeclipse.property.configuration.managers.PropertyManager;
+
+/**
+ * Handler for save property command
+ */
+public class SavePropertyHandler extends AbstractPropertyHandler {
+
+	private static final String DIALOG_TILE = "Upload platform config";
+	private static final String DIALOG_MESSAGE = "Are you sure, you want to upload selected properties to the platform?";
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	protected void execute(final Map<String, String> properties) {
+		if( MessageDialog.openConfirm(getActiveShell(), DIALOG_TILE, DIALOG_MESSAGE) ) {
+			final PropertyManager manager = new PropertyManager();
+			
+			if(manager.checkHacHealth()) {
+				properties.entrySet().stream().forEach(
```

Comment:
Please reformat . Move "forEach" to new line. Also you could try adjust it a little bit : 
```
 properties.entrySet().stream().
map(MapEtnry::getKey)
.forEach(manager::saveProperty)
```